python/pyspark/sql/dataframe.py

```diff
@@ -37,6 +37,7 @@
 from pyspark.sql.column import Column, _to_seq, _to_list, _to_java_column
 from pyspark.sql.readwriter import DataFrameWriter
 from pyspark.sql.streaming import DataStreamWriter
+from pyspark.sql.types import IntegralType
 from pyspark.sql.types import *
```

Comment:
Looks we don't need to import `IntegralType`.

Reply:
`IntegralType` is not in exposed list of `pyspark.sql.types`. Actually it is not used by any other places.

Reply:
I see ...